tck/features/uncategorized/ColumnNameAcceptance.feature

```diff
@@ -58,6 +58,7 @@ Feature: ColumnNameAcceptance
       | nOdEs( p ) |
     And no side effects
 
+  @withIntentionalStyleViolation
```

Comment:
I think the tag name is a bit long. I suggest some alternatives:

`@BadStyle`
`@StyleViolation`

I think the presence of the tag indicates the 'intentional' part to the reader and doesn't have to be included.

Or taking a different approach, more towards the effect of the tag rather than its definition:

`@NoStyleCheck`
`@SkipStyleCheck`

Reply:
I do not have strong opinion and change the tag name like three times before opening the PR. I agree it is fairly long. Out of you proposals I lean towards `@skipStyleCheck`. That is uses instructional language, like `@ignore`, and I can see a rule saying: 

- Instructional language for tags that make the API, etc. behave differently 
- Non-instructional language for tags that mark language feature used in a scenario (something, we do not yet have, but anticipate).

Concluding, I will change the tag to `@skipStyleCheck` and move it to a constant, as well. (Should do the same with `@ignore`, which I remember seen being use as string literal directly in code.)

Reply:
Sounds good. Personally I would go for an upper-case first letter, also for `@Ignore` but I'll concede that if you disagree.

Reply:
If you would like to keep the lower-case, perhaps a different scheme than camel case? Like `@skip_style_check` or `@skip-style-check`?

Reply:
I do not have a particular opinion other than that the existing tags @ignore and @allowCustomError start with a lower case letter. @ignore may also be used in custom tests outside the TCK but through the TCK API. Hence, changing this is sort of a breaking API change. We could allow also lower case for the old tags to not break the API and still changing to upper case. Is it worth the effort?

Reply:
No probably not. Consistency is probably more important here than stylism. Let's go with `@skipStyleCheck`. I'm coming around to it :)